stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -123,199 +136,59 @@ URLs, which facilitates WebSub discovery.
-
-public function main() {
-
-    var registrationResponse = websubHubClientEP->registerTopic("<TOPIC_URL>");
-    if (registrationResponse is error) {
-        log:printError("Error occurred registering topic: " + <string>registrationResponse.detail()?.message);
-    } else {
-        log:printInfo("Topic registration successful!");
-    }
-
-    // Make the publisher wait until the subscriber subscribes at the hub.
-    runtime:sleep(10000);
-
-    log:printInfo("Publishing update to remote Hub");
-    var publishResponse = websubHubClientEP->publishUpdate("<TOPIC_URL>", { "action": "publish", "mode": "remote-hub" });
-    if (publishResponse is error) {
-        log:printError("Error notifying hub: " + <string>publishResponse.detail()?.message);
-    } else {
-        log:printInfo("Update notification successful!");
-    }
-
-}
-```
-
-The `websub:SubscriptionClient` can be used by subscribers to send subscription and unsubscription requests explicitly.
-```ballerina
-import ballerina/log;
-import ballerina/websub;
-
-websub:SubscriptionClient websubHubClientEP = new("<HUB_URL>");
-
-public function main() {
-
+    
+    ```ballerina
+    websub:SubscriptionClient websubHubClientEP = new("<HUB_URL>");
+    
     // Send subscription request for a subscriber service.
     websub:SubscriptionChangeRequest subscriptionRequest = {
         topic: "<TOPIC_URL>", 
         callback: "<CALLBACK_URL>",
         secret: "<SECRET>"
     };
-
-    var subscriptionChangeResponse = websubHubClientEP->subscribe(subscriptionRequest);
-    if (subscriptionChangeResponse is websub:SubscriptionChangeResponse) {
-        log:printInfo("Subscription Request successful at Hub [" + subscriptionChangeResponse.hub 
-                        + "] for Topic [" + subscriptionChangeResponse.topic + "]");
-    } else {
-        log:printError("Error occurred with Subscription Request", err = subscriptionChangeResponse);
-    }
-
+    websub:SubscriptionChangeResponse|error subscriptionChangeResponse = websubHubClientEP->subscribe(subscriptionRequest);
+    
     // Send unsubscription request for the subscriber service.
     websub:SubscriptionChangeRequest unsubscriptionRequest = {
         topic: "<TOPIC_URL>",
         callback: "<CALLBACK_URL>"
     };
+    ebsub:SubscriptionChangeResponse|error subscriptionChangeResponse = websubHubClientEP->unsubscribe(unsubscriptionRequest);
+    ```
 
-    subscriptionChangeResponse = websubHubClientEP->unsubscribe(unsubscriptionRequest);
-    if (subscriptionChangeResponse is websub:SubscriptionChangeResponse) {
-        log:printInfo("Unsubscription Request successful at Hub [" + subscriptionChangeResponse.hub
-                + "] for Topic [" + subscriptionChangeResponse.topic + "]");
-    } else {
-        log:printError("Error occurred with Unsubscription Request", err = subscriptionChangeResponse);
-    }
-}
-```
+### Basic flow with WebSub
```

Comment:
Shall we move this section up? Otherwise, things come under the publisher and subscriber are hard to understand